test/index.js

```diff
@@ -172,15 +172,15 @@ test('TypeClass', function() {
   //  Bar :: TypeClass
   var Bar = Z.TypeClass('my-package/Bar', [Foo], hasMethod('bar'));
 
-  eq(Foo['@@type'], 'sanctuary-type-classes/TypeClass');
+  eq(Foo.constructor['@@type'], 'sanctuary-type-classes/TypeClass');
```

Comment:
`sanctuary` depends on `sanctuary-def`
`sanctuary-def` depends on `sanctuary-type-classes`
so  if we export it here we can use it in sanctuary-def and sanctuary